cli/init.ts

```diff
@@ -0,0 +1,97 @@
+import { Command } from "../deps/cliffy.ts"
+import * as fs from "../deps/std/fs.ts"
+import { parse } from "../deps/std/jsonc.ts"
+import * as path from "../deps/std/path.ts"
+import { detectVersion } from "../server/detectVersion.ts"
+
+export const init = new Command()
+  .description("Configure Capi in the current working directory")
+  .stopEarly()
+  .example("Configure Capi in the current directory", "capi init")
+  .action(runInit)
+
+async function runInit() {
+  const packageJsonPath = path.join(Deno.cwd(), "package.json")
+  if (await fs.exists(packageJsonPath)) return await runInitNode(packageJsonPath)
+  let denoJsonPath = path.join(Deno.cwd(), "deno.json")
+  if (await fs.exists(denoJsonPath)) return await runInitDeno(denoJsonPath)
+  denoJsonPath += "c"
+  if (await fs.exists(denoJsonPath)) return await runInitDeno(denoJsonPath)
+  throw new Error("Could not find neither a `package.json` nor `deno.json`/`deno.jsonc`.")
+}
+
+async function runInitNode(packageJsonPath: string) {
+  const packageJson = JSON.parse(await Deno.readTextFile(packageJsonPath))
+  assertManifest(packageJson)
+  const isTs = confirm("Are you using TypeScript?")
+  const devDependencies = packageJson.devDependencies ??= {}
+  if (isTs && !devDependencies["ts-node"]) {
+    const addTsNode = confirm(
+      `To define your net specs in TypeScript, we'll need to install the \`ts-node\` loader. Can we add it to your \`package.json\`?`,
+    )
+    if (addTsNode) {
+      devDependencies["ts-node"] = "*"
+    }
```

Comment:
if `ts-node` is not installed, `capi` can not be run

```
> capi init

(node:40288) ExperimentalWarning: Custom ESM Loaders is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

node:internal/process/esm_loader:42
      internalBinding('errors').triggerUncaughtException(
                                ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'ts-node' imported from /Users/parity/Code/4fun/harry-test/
```
after manually installing `npm i ts-node`, I get

```
    const isTs = confirm("Are you using TypeScript?");
                 ^

ReferenceError: confirm is not defined
```

Reply:
I addressed this feedback, with the exception of overriding / erroring on tsconfig specifics. Not sure how we should tackle this... a bit of a can of worms.

Reply:
Perhaps running Capi without `ts-node` so we don't need a `tsconfig.json`